.github/workflows/compound-java-dev.yml

```diff
@@ -133,3 +133,6 @@ jobs:
       javaVendor: ${{ inputs.javaVendor }}
     secrets:
       nvd-api-key: ${{ secrets.nvd-api-key }}
+
+  license-check:
+    uses: th2-net/.github/.github/workflows/license_check.yml@main
```

Comment:
```suggestion
    if: inputs.scanner-enabled
    uses: th2-net/.github/.github/workflows/license_check.yml@main
```

Reply:
If condition added to all files where license_check called